dungeon/test/dslToGame/TestDslFileLoader.java

```diff
@@ -0,0 +1,129 @@
+    private static final Path PATH_OF_THIRD_FILE =
+            Paths.get(".", "dslFileLoader", "testjar.jar", "scripts", "test.txt");
+
+    private static final Path PATH_TO_EMPTY_DNG = Paths.get(".", "dslFileLoader", "empty.dng");
+    private static final Path PATH_TO_TXT = Paths.get(".", "dslFileLoader", "test.txt");
+
+    @Test
+    public void processArguments_oneJar() throws IOException {
+        String[] args = {TEST_JAR_STRING};
+        Set<Path> paths = DslFileLoader.processArguments(args);
+        assertEquals(2, paths.size());
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_FIRST_FILE.normalize())));
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_SECOND_FILE.normalize())));
+        assertFalse(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_THIRD_FILE.normalize())));
+    }
+
+    @Test
+    public void processArguments_oneDSLFile() throws IOException {
+        String[] args = {SIMPLE_DNG_STRING};
+        Set<Path> paths = DslFileLoader.processArguments(args);
+        assertEquals(1, paths.size());
+        Path p = (Path) paths.toArray()[0];
+        assertTrue(p.endsWith(PATH_TO_DNG.normalize()));
+    }
+
+    @Test
+    public void processArguments_oneJarOneDSL() throws IOException {
+        String[] args = {TEST_JAR_STRING, SIMPLE_DNG_STRING};
+        Set<Path> paths = DslFileLoader.processArguments(args);
+        assertEquals(3, paths.size());
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_FIRST_FILE.normalize())));
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_SECOND_FILE.normalize())));
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_TO_DNG.normalize())));
+        assertFalse(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_THIRD_FILE.normalize())));
+    }
+
+    @Test
+    public void processArguments_mixed() throws IOException {
+        String[] args = {EMPTY_STRING, TEST_JAR_STRING, SIMPLE_DNG_STRING};
+        Set<Path> paths = DslFileLoader.processArguments(args);
+        assertEquals(4, paths.size());
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_FIRST_FILE.normalize())));
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_SECOND_FILE.normalize())));
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_TO_DNG.normalize())));
+        assertTrue(paths.stream().anyMatch(p -> p.endsWith(PATH_TO_EMPTY_DNG.normalize())));
+        assertFalse(paths.stream().anyMatch(p -> p.endsWith(PATH_OF_THIRD_FILE.normalize())));
+    }
+
+    @Test
+    public void processArguments_nonDSLFile() throws IOException {
+        String[] args = {TXT_STRING};
+        Set<Path> paths = DslFileLoader.processArguments(args);
+        assertEquals(0, paths.size());
+    }
+
+    @Test
+    public void fileToString() {
+        ClassLoader classLoader = getClass().getClassLoader();
+        File f = new File(classLoader.getResource(PATH_TO_DNG.toString()).getFile());
```

Comment:
```suggestion
        File f = new File(classLoader.getResource(PATH_TO_DNG.toString().replace("\\","/")).getFile());
```

... java path ist mit "/" und Paths.get gibt einen betriebssystem abhängig formattierten .... also "\" classloader geht nur von einen javapfad aus ...